react/wbooks/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import i18next from 'i18next';
```

Comment:
import `{ t }`
And use it directly
`<div className={styles.container}>{t('LOGIN:login')}</div>`